minutes/2016-09-09.md

```diff
@@ -0,0 +1,216 @@
+## 2016-09-09: General discussion
+
+Time: 14:00 UTC
+
+Hangout link: [](https://hangouts.google.com/call/v5olhwzpfzgzpoq5i3wthjpqpie)[https://hangouts.google.com/call/v5olhwzpfzgzpoq5i3wthjpqpie](https://hangouts.google.com/call/v5olhwzpfzgzpoq5i3wthjpqpie)
+
+**Attendees**
+
+*    Jonathan Helmus, Filipe, Michael, Ray, [Eric Dill](https://conda-forge.hackpad.com/ep/profile/yJqDqpPqJyz), Bjˆrn Gr¸ning, [Matt Craig](https://conda-forge.hackpad.com/ep/profile/yBvjHx0Ad3Y) (late)
```

Comment:
Yeah I guess Hackpad's markdown conversion struggles with unicode. Thanks for checking. Sorry your name got mangled, @bgruening. Will fix it.